### DISCUSSION TITLE
Installation Problems

### DISCUSSION THREAD

**Discussion by andrehs91** [symfony/symfony#45481] [Symfony Book: The Fast Track]
I'm starting to learn with "Symfony: The Fast Track", but I'm having problems installing the project.

**Command executed:** symfony new --version=5.4-1 --book guestbook

**Error:**
Getting Ready for the First Step of the Book
--------------------------------------------
[GIT] Check for not yet committed changes: [ OK ]
[GIT] Check Git un-tracked files: [ OK ]
[GIT] Removing Git ignored files (vendor, cache, ...): [ OK ]
[GIT] Resetting Git staged files: [ OK ]
[GIT] Removing un-tracked Git files: [ OK ]
[WEB] Adding .env.local: [ OK ]
[WEB] Stopping Docker Containers: Skipped for this step
[WEB] Stopping the Local Web Server: [ OK ]
[WEB] Stopping the Platform.sh tunnel: [ OK ]
[GIT] Checking out the step: [ OK ]
[SPA] Stopping the Local Web Server: Skipped for this step
[WEB] Installing Composer dependencies (might take some time): [ OK ]
[WEB] Adding .env.local: [ OK ]
[WEB] Starting Docker Compose: Skipped for this step
[WEB] Migrating the database: [ KO ]
**[ERROR] The version "latest" couldn't be reached, there are no registered migrations.**

**Comment by chyshyya** [comment]
Change `--version=5.4-1` to `--version=5.4`
Try it

**Reply by andrehs91** [reply]
It didn't work.
The https://github.com/the-fast-track/book-5.4 repository doesn't exist.
I also tried using "symfony new --version=6.0-1 --book guestbook" but the error was the same.
Should I do any additional configuration in the database?
